literature.bib

```diff
@@ -1545,6 +1571,32 @@ @inproceedings{FFB+:MODEVAR22
 	series = SPLC
 }
 
+@inproceedings{RBB+:ESECFSE22,
+	author = {Ramkisoen, Poedjadevie Kadjel and Businge, John and van Bladel, Brent and Decan, Alexandre and Demeyer, Serge and De Roover, Coen and Khomh, Foutse},
+	title = {{Pareco: Patched Clones and Missed Patches Among the Divergent Variants of a Software Family}},
+	year = 2022,
+	isbn = {9781450394130},
+	publisher = ACM,
+	address = NY,
+	doi = {10.1145/3540250.3549112},
+	booktitle = ESECFSE,
+	location = {Singapore, Singapore},
+	pages = {646--658},
+	numpages = {13},
+	keywords = {Bug-fixes, Clone detection, Clone&own, Effort duplication, Forking, Github, Social coding, Software family, Variants}
+}
+
+@inproceedings{HGP+:SPLC22,
```

Comment:
this one already exists on the main branch as HGP+:MODEVAR22

Reply:
Oops. This insertion seems to come from a messed up rebase conflict fix. I will remove this entry.